res/TensorFlowLiteRecipes/Net_ZeroDim_001/test.recipe

```diff
@@ -61,6 +72,16 @@ operand {
   name: "add"
   type: FLOAT32
   shape {
+    dim: 0
+    dim: 8
+    dim: 0
+    dim: 64
+  }
+  shape_signature {
+    dim: -1
+    dim: 8
+    dim: -1
+    dim: 64
```

Comment:
This is the shape of output. The output shape should match with the new shape inference logic in #13750.